src/Maestrano.Tests/Maestrano.Tests.Net40.csproj

```diff
@@ -86,12 +86,6 @@
     <WCFMetadata Include="Service References\" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Maestrano\Maestrano.csproj">
-      <Project>{3888e59d-f45b-405c-b8cd-039f0af0b7ce}</Project>
-      <Name>Maestrano</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
     <None Include="Support\Saml\Responses\response1.xml.base64" />
     <None Include="Support\Saml\Responses\response2.xml.base64" />
```

Comment:
I know Visual Studio is annoyting sometimes when he change the files without asking, could you please remove this as well from the PR ?